app/javascript/dashboard/components/widgets/conversation/conversationBulkActions/AgentSelector.vue

```diff
@@ -1,5 +1,16 @@
 <template>
   <div class="bulk-action__agents">
+    <div class="triangle">
+      <svg height="12" viewBox="0 0 24 12" width="24">
+        <path
+          d="M20 12l-8-8-12 12"
+          fill="var(--white)"
+          fill-rule="evenodd"
+          stroke="var(--s-50)"
+          stroke-width="1px"
+        ></path>
+      </svg>
```

Comment:
Let us use this SVG via fluent-icon pack

Reply:
This not an icon, its actually the caret/triangle for showing above popover which I custom made, also this has some extra attributes like stroke-width and stroke.

Should I add it icons, or I can make a separate component for this, `woot-popover`? 

@pranavrajs

Reply:
This is fine.